### PR TITLE
Debug Logger - add demisto formatter to rm sensitive url query params

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/31917

## Description
set the demisto handler logger to be int logger

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -4837,8 +4847,7 @@ def __init__(self):
             self.http_client_print = getattr(http_client, 'print', None)  # save in case someone else patched it already
             setattr(http_client, 'print', self.int_logger.print_override)
         self.handler = DemistoHandler()
```
Comment (glicht) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:4849]: There is no need for the DemistoFormatter. The fix should be that the handler should use the IntegrationLogger to write to the log and not directly to `demisto.debug()`. I think the following will fix the issue:
```suggestion
        self.handler = DemistoHandler(self.int_logger)
```

**Review by glicht — CHANGES_REQUESTED** [review]
See my comment. Please also add a unit test.

**Comment by glicht** [comment]
Also in DemistoHandler here: https://github.com/demisto/content/blob/5bf9794cebb6b0db21a02870baf8281ed171c63b/Packs/Base/Scripts/CommonServerPython/CommonServerPython.py#L4811 Need to change to use `call` and not `write`. So it should be:

```python
self.int_logger(msg)
```

**Review by yaakovi — APPROVED** [review]

**Review by glicht — APPROVED** [review]

File: Packs/Base/ReleaseNotes/1_5_3.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### CommonServerPython
+- Improved log formatter to remove sensitive URL query parameters.
```
Comment (amshamah419) [inline Packs/Base/ReleaseNotes/1_5_3.md:4]:
```suggestion
- Improved the log formatter to remove sensitive URL query parameters.
```

**Review by amshamah419 — APPROVED** [review]

**Pull request merged**